stanzas/venn-diagram/index.js

```diff
@@ -43,20 +43,22 @@ export default class VennStanza extends Stanza {
     }
 
     // get data
-    this.data = await this.getData();
+    const data = await this.getData();
+    this.data = transform(data);
```

Comment:
`Metastanza`　（`Stanza`ではなく）　を extend すると、　データが fetchされて、`this._data`　に保存されます。
今だと、同じデータが二度fetch になってしまいます。 
`this._data` にアクセスできるのは、 `Metastanza` クラスの `async render()`が関数が呼び出されたの後になります。
基本的に、 `Metastanza` をextend　する場合はは、　`async render()` ではなく、 `async renderNext()`というメソッドの中でデータ処理やレンダリングを行います。

```suggestion
    this.data = transform(this._data);
```